Makefile

```diff
@@ -88,7 +88,7 @@ GO_MOD_FLAGS ?= -mod=vendor
 endif
 
 .PHONY: build
-build: generate fmt vet
+build: fmt vet
```

Comment:
Please add a comment to the endpoint and the commit message that `make generate` needs to be called when adding a new CRD and will not happen by running `make build`.

Reply:
Please add the generated `bin` directory to `.gitignore` as part of this PR.

Reply:
Done